pulpanalytics/models.py

```diff
@@ -9,6 +9,15 @@ class System(models.Model):
     def __str__(self):
         return f"SystemID={self.system_id}, Created={self.created}"
 
+    class Meta:
+        constraints=[
+            models.UniqueConstraint(
+                "system_id",
+                models.functions.TruncDay("created"),
```

Comment:
It would not be that much easier. The migration would need to be the same. Just the uniqueness constraint would be slightly easier. And i'm not yet ready to throw away the data about seconds here.

Reply:
I can get behind that. Let's do this!